### PR TITLE
ReadMe Copy

### PR DESCRIPTION
**Pull request by Bedrock02**
#### What does this PR do?
- Establishes coding styles and structure
- Organizes Code of Conduct & Contribution

#### What are the relevant Trello Card stories?
[ Trello Card ](https://trello.com/c/YjWsOaYH)

#### Pull Request Type (check one)
- [ ] Feature
- [ ] Bug Fixes
- [x] Chore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,3 +87,63 @@ To deploy this Strapi instance you'll need:
...
+2. styles.js
+
+The purpose of having the styles outside of the component is to make it easier to revisit components and separate areas of concern.
+The purpose of naming the component [ComponentName].js is so that it is easier to search for, instead of searching for index.js.
+
+
+```
+src
+  - components
+    - NewComponent
```
Comment (Easybuoy) [inline README.md:116]: Folder names should be small letters

**Comment by samakintunde37** [comment]
@Easybuoy  Yeah, that would totally help clear some of the issues we currently have and those we'll be having later. I'll create an issue to this effect and work on it. Cool?

**Comment by Easybuoy** [comment]
> @Easybuoy Yeah, that would totally help clear some of the issues we currently have and those we'll be having later. I'll create an issue to this effect and work on it. Cool?

Let's get @Bedrock02 thought on this, maybe he might want to do it as part of this PR

**Comment by Bedrock02** [comment]
ya this makes sense @Easybuoy I like how #ios has it.

@samakintunde37 feel free to push those changes to this branch I can review later. Feel free to ping me as well.

**Comment by samakintunde37** [comment]
I'll get on it right away.

**Comment by samakintunde37** [comment]
I reckon we'll have to decide on a specific package manager since having generated lockfiles from both `yarn` and `npm` could cause issues. I get warnings from this in the terminal when I do it but I haven't any experience with such issues.

**Comment by samakintunde37** [comment]
@Easybuoy @Bedrock02 Wanna take a look at what we've got?

**Comment by Easybuoy** [comment]
> warnings from



> I reckon we'll have to decide on a specific package manager since having generated lockfiles from both `yarn` and `npm` could cause issues. I get warnings from this in the terminal when I do it but I haven't any experience with such issues.

Yeah true, what we can do to solve this is to delete both lock files and add them to the `.gitgnore` file

**Comment by Easybuoy** [comment]
> @Easybuoy @Bedrock02 Wanna take a look at what we've got?

Looks good, we need to add a link from the `README` to the `CONTRIBUTIONS`, because the `README` is the first sort of documentation people would see when they first get on the repository

**Comment by samakintunde37** [comment]
I'll update the Readme. While I'm doing that...

> delete both lock files

...shall I handle this or we do it in a separate PR?

**Comment by Easybuoy** [comment]
> I'll update the Readme. While I'm doing that...
> 
> > delete both lock files
> 
> ...shall I handle this or we do it in a separate PR?

You can do it on this PR

**Comment by Easybuoy** [comment]
> Looks good

If you're done, remove the `WIP` label @samakintunde37

**Comment by samakintunde37** [comment]
Great. I'm done.

**Comment by Bedrock02** [comment]
great work @samakintunde37 thanks

**Pull request merged**